rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -277,7 +276,7 @@ object Runtime {
       extraSystemProcesses: Seq[SystemProcess.Definition[F]]
   )(
       implicit
-      P: Parallel[F, M]
```

Comment:
Maybe now we can remove the `M[_]` in the type signature of `createWithEmptyCost_`?